### ISSUE TITLE
Keyboard navigation with bottom bar - release blocker

### ISSUE THREAD

**Issue by brianteeman** [joomla/backend-template#309]
If you try to navigate the screen with a keyboard then it is really easy to get lost as you can end up on **invisible** elements

For example when you tab from the editor then the next item in the tab order is the **toggle editor** but at my screen resolution it is completely obscured by the toolbar and the screen doesnt scroll to the tab. 

This is probably because it thinks the toggle editor button is  visible and doesnt understand that the toolbar is floating on top of it. 

The same issue is faced on any screen where a field is on the screen but obscured by the floating toolbar. It is never possible for a keyboard user to access that field. 

**obviously this is a major accessibility fail**

To showcase this I used chromelens to trace the keyboard path and recorded the screen

### Without the floating bottom tab bar
![good](https://user-images.githubusercontent.com/1296369/60201682-c449a380-9840-11e9-8885-387de730aa5f.gif)

### With the floating tab bar
![briken](https://user-images.githubusercontent.com/1296369/60201706-d1669280-9840-11e9-9d66-295419b3b00e.gif)

**Comment by brianteeman** [comment]
/cc @zwiastunsw I am sure you agree that this fails

![image](https://user-images.githubusercontent.com/1296369/60202259-18a15300-9842-11e9-9ae3-8513295e1e78.png)

**Comment by brianteeman** [comment]
Last night I had this reviewed and confirmed by the drupal core accessibility maintainer

**Comment by zwiastunsw** [comment]
Full consent. The order of the tabulation must be logical, in accordance with the source order and visual order. But the visual order must also be logical! It's not logical now.

**Comment by bembelimen** [comment]
Is this solved now with the toolbar at the top again?

**Comment by brianteeman** [comment]
it should be - will double check tomorrow before closing

**Comment by bembelimen** [comment]
Thx.

**Comment by brianteeman** [comment]
its ok now
